app/services/media-gallery/media-gallery.ts

```diff
@@ -154,23 +141,32 @@ export class MediaGalleryService extends Service {
     return new Request(url, { ...options, headers });
   }
 
+  private async fetchStockMedia(): Promise<{
+    audios: Array<IMediaGalleryFile>;
+    images: Array<IMediaGalleryFile>;
```

Comment:
I don't think there is a functional difference, but why the `Array<>`  syntax as opposed to just `IMediaGalleryFile[]`.  We have previously exclusively used the latter syntax.

Reply:
i think its when we switched from tslint to eslint but my IDE complains about the latter now